### ISSUE TITLE
Option to force CPU usage on machines without Vulkan on Windows

### ISSUE THREAD

**Issue by Wyrdgirn** [upscayl/upscayl#390]
### Checklist

- [X] I have checked that this issue isn't addressed in [the Wiki](https://github.com/upscayl/upscayl/wiki).
- [X] I have checked that this issue isn't addressed in any other issue or pull request.

### Description

The program does not work on machines without Vulkan, but it works if emulated, so it would be very helpful if they added the option to force the use of swiftshader.

I have tried deleting the **_vulkan-1.dll_** library and renaming a copy of **_vk_swiftshader.dll_** to **_vulkan-1.dll_**, then setting the GPU ID to 0 or leaving it blank and the app works correctly, I have already tried all the models and they work (although slow) even on an Intel Atom N455 with 2gb of ram and Windows 10 🤯

So I would consider an option in settings (or automatically) to force CPU usage only for those systems without Vulkan useful... and even if it supports it it can become a quick fix for those with driver issues, etc. 🤔

Oh, you can also post in the documentation how to convert the app to CPU only by renaming and replacing the libraries as I mentioned before if the option is not useful for you, then at least more people will have access to this great tool 😎👌 You decide 😉

Greetings 😃✌️

Oh, by the way, excuse my English, my native language is Spanish 😅

### Related Problem(s)

The app does not work if the machine does not support Vulkan

### Alternatives

Renaming the **_vk_swiftshader.dll_** library to **_vulkan-1.dll_**

**Comment by aaronliu0130** [comment]
🤯

**Comment by NayamAmarshe** [comment]
I think we can add this info to the wiki.

**Comment by aaronliu0130** [comment]
I assume the same can be done on Linux by renaming libvk_swiftshader.so to libvulkan.so.1 . Not sure about Mac.

Maybe we should make a separate distribution for such builds?

**Comment by NayamAmarshe** [comment]
No, I think it'd be too much work for what's basically a smart hack.

I think it'd be better to let hacks stay in the wiki.

**Comment by NayamAmarshe** [comment]
Added here: https://github.com/upscayl/upscayl/wiki/Misc

**Comment by danielrode** [comment]
> I assume the same can be done on Linux by renaming libvk_swiftshader.so to libvulkan.so.1 . Not sure about Mac.
> 
> Maybe we should make a separate distribution for such builds?

When I try this on Void Linux using the AppImage version of Upscayl, I get an error:

_"BACKEND REPORTED:  image upscayl:  /tmp/.mount_upscay64N2F7/resources/bin/upscayl-bin: symbol lookup error: /tmp/.mount_upscay64N2F7/resources/bin/upscayl-bin: undefined symbol: vkAllocateDescriptorSets"_

**Comment by aaronliu0130** [comment]
Sounds like your driver really hates Vulkan. Sorry.
